README.md

```diff
@@ -4,28 +4,35 @@
 [![Tests](https://github.com/tiobe/tics-github-action/actions/workflows/test.yml/badge.svg)](https://github.com/tiobe/tics-github-action/actions/workflows/test.yml)
 [![CodeQL](https://github.com/tiobe/tics-github-action/actions/workflows/codeql.yml/badge.svg)](https://github.com/tiobe/tics-github-action/actions/workflows/codeql.yml)
 
-The TICS Github action integrates TICS Client analysis to measure your code quality. The incorporated Quality gating feature enables you to analyze and decorate pull requests.
+The TICS Github action integrates the [TICS code quality framework](https://www.tiobe.com/products/tics/) into your workflows. This allows you to effectively measure and monitor the software code quality of all your projects and its branches.
+The incorporated `Quality Gating` feature can be used for `Pull Request` and `Commit` approvals. It can also decorate your pull request so the quality data is easily available. 
+Furthermore, the changed files are annotated with findings from the TICS analysis, so it is clear where the issues are and thus where they need to be addressed.
 
-## Before you start
+There are two types of analysis modes available:
+- Reference runs using TICS QServer (See section QServer). Reference points with code quality metric data are created that are used for the qualification runs. Intended for branches where other branches are derived from and will be merged back to (Like main).

```

Comment:
> Intended for branches where other branches are derived from and will be merged back to (Like main).

This reads a bit confusing to me. Perhaps we can incorporate the term 'base branch', which seems to be the conventional name in GitHub for 'the branch PRs are merged to'. For example: 'Intended for base branches of pull requests, e.g. `main`.' Similarly, we could refer to the 'branch to be merged' as 'head branch' of a PR